File: not_required/temp.R
#!/usr/bin/Rscript --vanilla
setwd("C:/xampp/htdocs/webtool")
args <- commandArgs(TRUE)
cols <- args[1]
fname <- args[2]
x <- rnorm(cols,0,1)
fname = paste(fname, "png", sep = ".")
png(filename=fname, width=500, height=500)
hist(x, col="red")
dev.off()
#Terminal script: "C:/Program Files/R/R-4.2.0/bin/x64/Rscript.exe" "C:/xampp/htdocs/webtool/temp.R" 6 "temp"

File: not_required/temp.php
<html>
  <head>
    <title>PHP and R Integration Sample</title>
  </head>
  <body>
    <div id="r-output" id="width: 100%; padding: 25px;">
    <?php
      $filename = "samplefile".rand(1,100);
      $exe = '"C:/Program Files/R/R-4.3.1/bin/x64/Rscript.exe" "C:/xampp/htdocs/webtool/temp.R" 6 "temp"';
      exec($exe);
      echo $exe;
      echo "<img src ='temp.png'> ";
	  $target_dir = $_SERVER['DOCUMENT_ROOT'] . "/webtool/uploads/";
	  echo $target_dir;
    ?>
      
    </div>
  </body>
</html>